runtime/services/catalog/migrator/metricsviews/metrics_views.go

```diff
@@ -67,30 +64,25 @@ func (m *metricsViewMigrator) Validate(ctx context.Context, olap drivers.OLAPSto
 		fieldsMap[field.Name] = field
 	}
 
-	if _, ok := fieldsMap[mv.TimeDimension]; !ok {
-		return migrator.CreateValidationError(catalog.Path, TimestampNotFound)
+	// if a time dimension is selected it should exist
+	if mv.TimeDimension != "" {
+		if _, ok := fieldsMap[mv.TimeDimension]; !ok {
+			return migrator.CreateValidationError(catalog.Path, TimestampNotFound)
+		}
 	}
 
 	var validationErrors []*runtimev1.ReconcileError
 
 	for i, dimension := range mv.Dimensions {
-		err := validateDimension(ctx, model, dimension)
-		if err != nil {
+		if _, ok := fieldsMap[dimension.Name]; !ok {
```

Comment:
The `fieldsMap[name]` is a case sensitive comparison, but we consider column names case insensitive

Reply:
good catch

Reply:
updated